### PR TITLE
Langchain intro - Example - example_selectors fix + Vertex AI MatchingEngine GCS url fix for index creation

### PR DESCRIPTION
**Pull request by arindam-b**
Fix - Vertex AI Matching engine requires GCS url for init_index. The GCS URL was wrong which is corrected and a small typo.

Fix - embedded links in the notebook for example_selectors, text splitter, summarization chain and chain creation in Langchain intro.

### CODE REVIEW

**Comment by polong-lin** [comment]
Hi @arindam-b , thanks for fixing these links! Before we can proceed with merging, you would need to accept and sign the CLA: 
https://github.com/GoogleCloudPlatform/generative-ai/pull/104/checks?check_run_id=14908796396

**Comment by arindam-b** [comment]
Hi @polong-lin, I have accepted and signed the CLA, thanks.

File: language/examples/langchain-intro/intro_langchain_palm_api.ipynb [hunk 1]
```diff
@@ -1025,9 +1025,9 @@
    "source": [
     "### **Text Splitters**\n",
     "\n",
-    "[Text Splitters](https://python.langchain.com/en/latest/modules/indexes/text_splitters.html) are a way to deal with input token limits of LLMs by splitting text into chunks.\n",
+    "[Text Splitters](https://python.langchain.com/docs/modules/data_connection/document_transformers/) are a way to deal with input token limits of LLMs by splitting text into chunks.\n",
```
Comment (polong-lin) [inline language/examples/langchain-intro/intro_langchain_palm_api.ipynb:1028]: Looks like this link for Document Transformers doesn't match the text: "Text Splitters"?

**Review by polong-lin — CHANGES_REQUESTED** [review]

File: language/examples/langchain-intro/intro_langchain_palm_api.ipynb (hunk 1 above)
Comment (arindam-b) [reply language/examples/langchain-intro/intro_langchain_palm_api.ipynb:1028]: Thanks for the comment. If you look at the langchain documents, earlier "Indexes" were part of the Modules and "Text Splitter" was under "Indexes". Now, the structure is: "Modules" -> "Data connection" -> "Document transformers" -> "Text splitters". So the link I have specified describes the Text splitters only, which falls under Transformers. You can find the image below.

screen-shot


It is also worth noting, the langchain picture in the notebook with different core components needs to be updated, for which I will raise a new PR. Let me know your comments.

**Comment by arindam-b** [comment]
@polong-lin: Another modification was added, this time to the Vertex AI Matching engine index creation. That's why I have modified the PR title and added some details. Could you please review it? Thanks.

**Review by polong-lin — APPROVED** [review]
LGTM -- thank you for spotting these issues, and fixing the code/links!

**Pull request merged**